initfiles/bin/init_thor

```diff
@@ -133,18 +110,12 @@ thorpid=0
 
 while [[ 1 ]]; do
     # update slaves file in case state of environment has been altered since last run
-    if [[ "$processperslave" = "true" ]]; then
-        daliadmin server=$DALISERVER dfsgroup ${groupName} slaves
-    else
-        daliadmin server=$DALISERVER clusternodes ${groupName} slaves
-    fi
+    daliadmin server=$DALISERVER clusternodes ${groupName} uslaves.start
     errcode=$?
     if [[ 0 != ${errcode} ]]; then
     log "failed to lookup dali group for $groupName"
         exit 1
     fi
-    makethorgroup
-    sort $instancedir/slaves | uniq > $instancedir/uslaves.start
```

Comment:
your uslaves.start file that you create on line 113 isn't sorted and uniq.  When we go to do a diff on it against the hosts.uniq file on line 163 it can fail.  I would suggest changing line 113 to 'slaves' instead of 'uslaves.start' and then keeping line 147.